stl/CMakeLists.txt

```diff
@@ -480,16 +480,16 @@ function(add_stl_dlls D_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIO
 
     # import library 'statics'
     add_library(msvcp${D_SUFFIX}_implib_objects OBJECT ${IMPLIB_SOURCES})
-    target_compile_definitions(msvcp${D_SUFFIX}_implib_objects PRIVATE "_DLL;${THIS_CONFIG_DEFINITIONS}")
-    target_compile_options(msvcp${D_SUFFIX}_implib_objects PRIVATE "${THIS_CONFIG_COMPILE_OPTIONS};$<$<COMPILE_LANGUAGE:CXX>:/EHsc>") # No /GL!

```

Comment:
`$<COMPILE_LANGUAGE:CXX>` was added during the alias object refactoring - why is it unnecessary now?

Reply:
it's now it's own library. Looks like we forgot to drop this badness when we made that change.

Reply:
it's own object library to be clear